core/store/src/db.rs

```diff
@@ -386,6 +391,18 @@ impl Database for RocksDB {
                 DBOp::Insert { col, key, value } => unsafe {
                     batch.put_cf(&*self.cfs[col as usize], key, value);
```

Comment:
I think we should also add assert here to check that the column is not write_once.

Reply:
Here, we ended up saying: "DBTx is private and has no assertions, but StoreUpdate has all of the assetions":

https://github.com/near/nearcore/pull/6674